src/frontend/js/components/ContractFrame/index.spec.tsx

```diff
@@ -0,0 +1,97 @@
+import { QueryClientProvider } from '@tanstack/react-query';
```

Comment:
In commit [f27f0b9b8751bfac46cce909aa2507bade13d5ea](https://github.com/openfun/richie/commit/f27f0b9b8751bfac46cce909aa2507bade13d5ea) message

```diff
-ContractFrame is a generic comopnent that can be called from anywhere.
+ContractFrame is a generic component that can be called from anywhere.
```